staker/stateless_block_validator.go

```diff
@@ -69,6 +70,7 @@ type TransactionStreamerInterface interface {
 
 type InboxReaderInterface interface {
 	GetSequencerMessageBytes(ctx context.Context, seqNum uint64) ([]byte, error)
+	FetchHotShotCommitment(blockHeight uint64) (espresso.Commitment, error)
```

Comment:
3. We can add a method like this to fetch the desired hotshot commitment in the validation pipeline, which will end up in the ValidationInput passed to the WASM layer.